src/components/DynamicFormContainer/DynamicForm/AttachmentDropzone/FilesInfo/FilesInfo.tsx

```diff
@@ -2,34 +2,60 @@ import prettyBytes from "pretty-bytes";
 import React, { FunctionComponent } from "react";
 import { FileUploadType } from "../../../../../types";
 import { Paperclip, X } from "react-feather";
+import pdf from "./pdf.svg";
+import jpg from "./jpg.svg";
+import doc from "./doc.svg";
+import csv from "./csv.svg";
+import png from "./png.svg";
+import txt from "./txt.svg";
 
 interface FilesInfoType {
   filesInfo: FileUploadType[];
   removeFile: (index: number) => void;
 }
 
+export const getExtension = (fileType: string): { hasIcon: boolean; fileType: string } => {
+  const iconName = fileType.split("/")[1] || "";
+  if (iconName == "pdf") return { hasIcon: true, fileType: pdf };
+  else if (iconName === "jpeg") return { hasIcon: true, fileType: jpg };
+  else if (iconName === "vnd.openxmlformats-officedocument.wordprocessingml.document")
+    return { hasIcon: true, fileType: doc };
+  else if (iconName === "csv") return { hasIcon: true, fileType: csv };
+  else if (iconName === "png") return { hasIcon: true, fileType: png };
+  else if (iconName === "plain") return { hasIcon: true, fileType: txt };
+  else return { hasIcon: false, fileType: iconName };
+};
```

Comment:
there's a few problems with this:
  - abit hard to read
  - i see that u r checking by breaking up mimeType actually, if that's the case, could check for mimeType directly?

would this be better?

```suggestion
interface ExtensionIconProps {
  src: string;
}

const ExtensionIcon: FunctionComponent<ExtensionIconProps> = ({ src }: ExtensionIconProps) => {
  return <img src={src} className="flex items-center justify-center mr-4" />;
};

const getExtensionIcon = (mimeType: string): React.ReactNode => {
  switch (true) {
    case mimeType === "text/csv":
      return <ExtensionIcon src={csv} />;
    case mimeType === "application/vnd.openxmlformats-officedocument.wordprocessingml.document":
      return <ExtensionIcon src={doc} />;
    case mimeType === "image/jpeg":
      return <ExtensionIcon src={jpg} />;
    case mimeType === "image/png":
      return <ExtensionIcon src={png} />;
    case mimeType === "application/pdf":
      return <ExtensionIcon src={pdf} />;
    case mimeType === "text/plain":
      return <ExtensionIcon src={txt} />;
    default:
      return (
        <div className="rounded-full bg-grey-lighter h-12 w-12 flex items-center justify-center mr-2">
          <Paperclip />
        </div>
      );
  }
};


```

then in the component u can use `{getExtensionIcon(type)}`